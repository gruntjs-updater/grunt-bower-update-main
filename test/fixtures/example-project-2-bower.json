{
  "name": "example-project-2",
  "main": [],
  "version": "0.0.1",
  "authors": [
    "Matthew J. Martin <matthew.mar10@gmail.com>"
  ],
  "description": "Example Project 2",
  "license": "MIT",
  "homepage": "http://www.example.com",
  "private": true,
  "ignore": [
    "**/should-be-excluded.js",
    "node_modules",
    "bower_components",
    "test",
    "tests"
  ],
  "dependencies": {
    "es5-shim": "~4.0.3",
    "json3": "~3.3.2"
  },
  "devDependencies": {
    "bootstrap": "~3.2.0"
  }
}