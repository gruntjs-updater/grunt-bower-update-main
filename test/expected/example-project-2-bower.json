{
  "name": "example-project-2",
  "main": [
    "test/fixtures/example-project-2-files/example-directory/example-file-4.js",
    "test/fixtures/example-project-2-files/example-directory/example-sub-directory/example-sub-sub-directory/example-file-5.js",
    "test/fixtures/example-project-2-files/example-file-1.js",
    "test/fixtures/example-project-2-files/example-file-2.js",
    "test/fixtures/example-project-2-files/example-file-3.js"
  ],
  "version": "0.0.1",
  "authors": [
    "Matthew J. Martin <matthew.mar10@gmail.com>"
  ],
  "description": "Example Project 2",
  "license": "MIT",
  "homepage": "http://www.example.com",
  "private": true,
  "ignore": [
    "**/should-be-excluded.js",
    "node_modules",
    "bower_components",
    "test",
    "tests"
  ],
  "dependencies": {
    "es5-shim": "~4.0.3",
    "json3": "~3.3.2"
  },
  "devDependencies": {
    "bootstrap": "~3.2.0"
  }
}